configs/dc2.yaml

```diff
@@ -0,0 +1,51 @@
+facility_name: Rubin-LSST
+obs_collection: LSST.DC2
```

Comment:
After some discussion in the science team, I think we're going to want to have separate `obs_collection` values for different pipeline outputs.  Is that configurable here, by moving `obs_collection:` down into the type-dependent section?

Reply:
There is only one global `obs_collection`, I'll add per-dataset type option which overrides global one.

Reply:
With the latest commit you can now specify `obs_collection` both at top level and per-dataset type.